acorn/src/expression.js

```diff
@@ -571,6 +584,14 @@ pp.parseParenExpression = function() {
   return val
 }
 
+pp.checkCommaAfterRest = function() {
```

Comment:
Can you explain what you use this for? It seems TypeScript also disallows a comma after a rest argument.

Reply:
Comma will be allowed in the declare function.
```
declare function test(a, ...b,)
```

Reply:
It seems reasonable to have a separate parser for function types anyway, so I don't think we should add a hook for this in the regular function parsing logic.

Reply:
Yeah, its not necessary to split the function in regular js parser. Anyway that will be great if you accept to add this hook. XD

Reply:
I asked to remove this, but it's still here.